### PR TITLE
Doc quickguide update

### PR DESCRIPTION
**Pull request by peaceshi**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
```
Comment (lhmouse) [inline .vscode/extensions.json:3]: Please remove comments. JSON does not support comments.

File: doc/quick-guide.md [hunk 2]
```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
...
-AND operator trims the longer string, and produces a result of the same
-length as the shorter one; while the byte-wise OR and XOR operators treat
-'missing information' as zeroes (which means to copy from the longer string),
-and produce a result of the same length as the longer one.
+byte-wise operations:
+
+``` node
+#1:1> "12345678"
+* running 'expression #1'...
+* result #1: string(8) "12345678";
```
Comment (lhmouse) [inline doc/quick-guide.md:799]: This piece of code makes little sense.

File: doc/quick-guide.md [hunk 3]
```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
...
+```
+
+Shifting strings to the left (`<<`) has the effect to fill zero bytes in the
+right;
+Shifting strings to the right (`>>`) has the effect to remove characters from
```
Comment (lhmouse) [inline doc/quick-guide.md:804]: If the line above ends with a semicolon, then the next sentence shall start with a lowercase letter, and shall not start on its own line.

The parentheses are misleading; they look as if `<<` and `>>` are the only shift operators. These sentences apply to all shift operators.

File: doc/quick-guide.md [hunk 4]
```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
...
+#2:1> "12345678" >> 1
+* running 'expression #2'...
+* result #2: string(7) "1234567";
+
+#3:1> "12345678" << 1
+* running 'expression #3'...
+* result #3: string(9) "12345678\0";
+```
+
+Logical shift operators (`>>>`and `<<<`) also fill zero bytes in the other
```
Comment (lhmouse) [inline doc/quick-guide.md:817]: We don't enclose appositives in parentheses.

File: doc/quick-guide.md [hunk 5]
```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
...
+#3:1> "2345" & "1234"
+* running 'expression #3'...
+* result #3: string(4) "0204";
+
+#4:1> "2345" & "12345678"
+* running 'expression #4'...
+* result #4: string(4) "0204";
+```
+
+while the byte-wise OR (`|`) and XOR (`^`) operators treat 'missing
```
Comment (lhmouse) [inline doc/quick-guide.md:856]: This paragraph shall start with a capital letter.

File: doc/quick-guide.md [hunk 6]
```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
...
+* result #4: string(8) "33755678";
+
+#5:1> "23450000" | "12345678"
+* running 'expression #5'...
+* result #5: string(8) "33755678";
+
+#6:1> "02345000" | "12345678"
+* running 'expression #6'...
+* result #6: string(8) "12345678";
+```
```
Comment (lhmouse) [inline doc/quick-guide.md:884]: I'd say these three pieces of code had better be merged into a single one

File: .vscode/extensions.json (hunk 1 above)
Comment (peaceshi) [reply .vscode/extensions.json:3]: this is supported by vscode default.

**Pull request merged**